### PR TITLE
Adding StringValidatorAttribute tests

### PR DESCRIPTION
**Pull request by dhoehna**

### CODE REVIEW

File: src/System.Configuration.ConfigurationManager/tests/System/Configuration/StringValidatorAttributeTests.cs [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+        [Fact]
+        public void InvalidCharacters_GetAndSet()
+        {
+            StringValidatorAttribute attribute = new StringValidatorAttribute();
+            attribute.InvalidCharacters = "_-";
+            Assert.Equal("_-", attribute.InvalidCharacters);
+
+        }
+
```
Comment (danmosemsft) [inline src/System.Configuration.ConfigurationManager/tests/System/Configuration/StringValidatorAttributeTests.cs]: Nit; extra blank lines

Comment (danmosemsft) [inline src/System.Configuration.ConfigurationManager/tests/System/Configuration/StringValidatorAttributeTests.cs]: Looks good otherwise -- thanks!

Comment (dhoehna) [inline src/System.Configuration.ConfigurationManager/tests/System/Configuration/StringValidatorAttributeTests.cs]: @danmosemsft How do you find these?

Comment (danmosemsft) [inline src/System.Configuration.ConfigurationManager/tests/System/Configuration/StringValidatorAttributeTests.cs]: They show up in the github diff view, nothing special. Or do you mean how do I have such eagle eyes? I don't know, done it a lot, naturally picky person? :)

Extra spaces don't show up here though. You'd have to do "git diff" then I think by default it highlights them. I obviuosly don't bother doing that on other people's changes.

Comment (dhoehna) [inline src/System.Configuration.ConfigurationManager/tests/System/Configuration/StringValidatorAttributeTests.cs]: More about how you have eagle eyes.  I mean, you caught typos in my method names.

Comment (danmosemsft) [inline src/System.Configuration.ConfigurationManager/tests/System/Configuration/StringValidatorAttributeTests.cs]: Thousands of code reviews? @stephentoub is more eagle eyed than me...

**Pull request merged**